custom-ops-matrix-multiplication/operations/matrix_multiplication.mojo

```diff
@@ -968,14 +968,15 @@ struct MatrixMultiplication[algorithm: StringLiteral]:
                     block_dim=(NUM_THREADS),
                 )
             elif algorithm == "tensor_core":
+                constrained[""]
```

Comment:
Is this left over from debugging?

Reply:
correct- was planning on constraining the inputs dtype to float32 (to make the tensor_core code path happy) but realized it's probably more robust to come up with a mapping of dtype to supported MMA instructions to generalize the examples to other input types.

Reply:
removed.   
LMK what you think of the mapping above, happy to contribute it in a separate PR.

Reply:
Thanks, let me check and see what we want to do to generalize this over multiple datatypes.